### DISCUSSION TITLE
Perfomance improvement on Raspberry pi 4 with faster SD card or USB 3 sitck ?

### DISCUSSION THREAD

**Discussion by Striker62** [photobooth-app/photobooth-app#305] [Q&A] [answered]
Hi,
would it make sence in terms of performance to use a faster SD card or run the system on an fast USB stick ?
Any experiance ?
Thanks

**Comment by mgineer85** [comment]
I usually use extreme pro cards, which I think are on the faster side. I do not have experience with slower cards.
It could improve times to capture a bit, but apart from this the disk is not used very much.

**Reply by Striker62** [reply]
perfect! thank you.
Than I can stop investigating .

What Raspberry pi have you in use Pi 4 with ? ram ?

**Reply by mgineer85** [reply, marked as answer]
I think I have some with 4gb and some with 8gb. Both should be fine.
